openpype/modules/kitsu/utils/update_op_with_zou.py

```diff
@@ -162,14 +179,32 @@ def update_op_assets(
         )
         if visual_parent_doc_id is None:
             # Find root folder doc
-            root_folder_doc = dbcon.find_one(
-                {
-                    "type": "asset",
-                    "name": entity_parent_folders[-1],
-                    "data.root_of": substitute_item_type,
-                },
-                ["_id"],
+            root_folder_docs = get_assets(
```

Comment:
As the function is `get_assets`, `root_folder_docs` is not explicit enough, if you want to mention that they are `docs`, use `root_folder_assets_docs`.

Reply:
I just followed naming with `root_folder_doc`.